analytics/management/commands/populate_analytics_db.py

```diff
@@ -48,17 +34,14 @@ def generate_fixture_data(self, stat: CountStat, business_hours_base: float,
 
     def handle(self, *args: Any, **options: Any) -> None:
         do_drop_all_analytics_tables()
-        # I believe this also deletes any objects with this realm as a foreign key
-        Realm.objects.filter(string_id='analytics').delete()
 
         installation_time = timezone_now() - timedelta(days=self.DAYS_OF_DATA)
         last_end_time = floor_to_day(timezone_now())
-        realm = Realm.objects.create(
-            string_id='analytics', name='Analytics', date_created=installation_time)
-        shylock = self.create_user('shylock@analytics.ds', 'Shylock', True, installation_time, realm)
-        stream = Stream.objects.create(
-            name='all', realm=realm, date_created=installation_time)
-        Recipient.objects.create(type_id=stream.id, type=Recipient.STREAM)
+        realm = get_realm('zulip')
+        shylock = get_user("shylock@zulip.com", realm)
```

Comment:
Creating shylock like other users doesn't create any problem in stats so far, so is it safe to left this as it is now or should I update it's fields here manually.

Reply:
@rishig can you clear this doubt?

Reply:
Creating shylock like other users is fine.